envault/cli.py

```diff
@@ -33,13 +35,56 @@ def get_secrets(server, secret, token):
 def cli():
     pass
 
+@cli.command("init")
+@click.option("-profile", help="profile for the config")
+def init(profile):
+    if profile is None:
+        profile = 'default'
+    """ Initialize envault config with vault server, token and secrets path """
+    click.echo("Enter the server, token and path to vault secrets")
+    vault_server = click.prompt('Vault Server', type=str)
+    vault_token = click.prompt('Vault Token', type=str)
+    vault_secret_path = click.prompt('Path to vault secret', type=str)
+
+    click.echo("server %s token %s path %s" %(vault_server, vault_token, vault_secret_path))
```

Comment:
You can use string formatting to better convey the variables used.
Also format it better so that the output looks good.

```
'''
Server: {vault_server}
Token: {token}
Path: {vault_secret_path}
'''.format(
 vault_server=vault_server, 
 vault_token=vault_token, 
 vault_secret_path=vault_secret_path
)
```